### PR TITLE
Related to #41: allow changelogs to be uploaded as part of metadata uploads

### PR DESCRIPTION
**Pull request by lacostej**

### CODE REVIEW

File: lib/supply/uploader.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@ def perform_upload
           upload_metadata(language, listing) unless Supply.config[:skip_upload_metadata]
           upload_images(language) unless Supply.config[:skip_upload_images]
           upload_screenshots(language) unless Supply.config[:skip_upload_screenshots]
+
```
Comment (asfalcone) [inline lib/supply/uploader.rb]: Remove extra newline.

**Comment by asfalcone** [comment]
@lacostej Thanks, this is great. I had 1 tiny comment about the extra new line. Once that is fixed up, I'll merge this :trophy:

**Comment by lacostej** [comment]
@asfalcone removed extra new line

**Comment by asfalcone** [comment]
@lacostej Thanks! 💯

**Pull request merged**